ci/kind-load-docker-images.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+"""
+Run `kind load docker-image <image:tag>` on all the docker images within
+values.yaml that is available locally on the host as first verified with `docker
```

Comment:
Consider adding the command line options to the docstring.

Reply:
Added it as a TODO!

Reply:
Done